roles/openshift_on_openstack/tasks/main.yml

```diff
@@ -330,12 +330,16 @@
   set_fact:
     openshift_install_log: "{{ log_directory }}/openshift_install.log"
 
+# Register pbench default tool-set
+- name: Register pbench toolset
+  shell: pbench-register-tool-set
+
 - name: Install OpenShift block
   block:
     # Run the Ansible playbook that installs OpenShift.
     - name: Installing OpenShift on OpenStack
       # Using two inventory flags here as the README instructs.
-      shell: "{{ ansible_playbook }} -vvv --user openshift -i inventory -i {{ inventory_py }} {{ openshift_openstack_dir }}/openshift-cluster/install.yml 2>&1 >> {{ openshift_install_log }}"
+      shell: pbench-user-benchmark --pbench-post='/usr/local/bin/pbscraper -i $benchmark_results_dir/tools-default -o $benchmark_results_dir; ansible-playbook -vv -i {{ pbcompare_inventory }} {{ pbcompare_playbook }} -e new_file='$benchmark_results_dir/out.json'' -- {{ ansible_playbook }} -vvv --user openshift -i inventory -i {{ inventory_py }} {{ openshift_openstack_dir }}/openshift-cluster/install.yml 2>&1 >> {{ openshift_install_log }}
```

Comment:
@chaitanyaenr Will removing the quotes work here? I am not sure the variables will be resolved.

Reply:
Feel free to use the ">" character here if you want to separate these on different lines.